src/main/java/com/boeing/hw1/ShowDate.java

```diff
@@ -34,6 +34,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
 		out.println(new Date());
 
 		out.println("<br> Jieun is writing hello world!");
+		out.println("<br>Branch Test - jieun");
 
 		out.println("New Hello World.");
 		out.println("New Hello World.");

```

Comment:
Did you really mean to repeat the previous line?  I think we should take that out.